Resources/Prototypes/_Sunrise/FleshCult/hands_mods.yml

```
@@ -1,137 +1,137 @@
-    - type: AmmoCounter
-    - type: Appearance
-    - type: RechargeBasicEntityAmmo
-      rechargeCooldown: 2
-      rechargeSound: /Audio/_Sunrise/FleshCult/spike_gun_reload.ogg
-    - type: Unremoveable
-      deleteOnDrop: true
-    - type: BasicEntityAmmoProvider
-      proto: BulletSpike
-      capacity: 10
-      count: 5
+- type: entity
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)